timesketch/app.py

```diff
@@ -181,6 +196,8 @@ def load_user(user_id):
     # Setup CSRF protection for the whole application
     CSRFProtect(app)
 
+    
+
     return app
```

Comment:
```suggestion
    CSRFProtect(app)

    return app
```

Removing empty lines